test/cluster/haproxy.cfg

```diff
@@ -0,0 +1,46 @@
+global
```

Comment:
move this file into examples/cluster/haproxy.cfg instead

Reply:
Maybe also test i/o with `zb` also.

Reply:
Also, give a separate name for this clustering setup - "stateless zot with shared reliable storage". There could be others and then we have to scramble.

Reply:
fixed